pyvex/lifting/libvex.py

```diff
@@ -16,12 +16,25 @@
 VEX_MAX_BYTES = 5000
 
 
+class VexRegisterUpdates:
+    VexRegUpd_INVALID = 0x700
+    VexRegUpdSpAtMemAccess = 0x701
+    VexRegUpdUnwindregsAtMemAccess = 0x702
+    VexRegUpdAllregsAtMemAccess = 0x703
+    VexRegUpdAllregsAtEachInsn = 0x704
+    VexRegUpdLdAllregsAtEachInsn = 0x705
+
+
 class LibVEXLifter(Lifter):
 
-    __slots__ = ()
+    __slots__ = ('_px_control', )
 
     REQUIRE_DATA_C = True
 
+    def __init__(self, arch, addr):
+        super().__init__(arch, addr)
+        self._px_control = ffi.new("VexRegisterUpdates *")
```

Comment:
We don't ever have any need to do anything other than pass this value in from python, so why are we passing a pointer here? We can just have pyvex.c do whatever referencing needs to be done.